BLE_SecurityAndPrivacy/source/main.cpp

```diff
@@ -16,7 +16,7 @@
 
 #include <events/mbed_events.h>
 #include "ble/BLE.h"
-#include "mbed-os-ble-utils/pretty_printer.h"
+#include "pretty_printer.h"
```

Comment:
why is this needed ? Or should it be as it was and we add additional include path to CMake where utils.lib is provided.

Rethorical questions if our external libraries should use same dir structure (providing directory in the root with the same name as library that would allow to use `name_library/include.h`)